### PR TITLE
Prevent pkg from being ran as root

### PR DESCRIPTION
**Pull request by Damaj301damaj-lol**
Thought that it might be necessary

### CODE REVIEW

**Comment by 2096779623** [comment]
```bash
.../files/home # pkg cl
Ability to run this command as root has been disabled permanently for safety purposes.
.../files/home #
```
Already added.

https://github.com/termux/termux-packages/blob/ce2d57424c1ed473c4fe66ab6550ab7d15a6bf10/packages/apt/0010-prevent-usage-as-root.patch#L24

**Comment by agnostic-apollo** [comment]
>Thanks for PR, we already added it.

That gets engaged way too late and not for `pacman`. If running `pkg` as root, then mirror selection would also create `sources.list` files with `root` ownership if they don't exist.

File: scripts/pkg.in [hunk 1]
```diff
@@ -1,6 +1,12 @@
 #!/bin/bash
 set -eu
 
+# Root Check
+case $(id -u) in
```
Comment (agnostic-apollo) [inline scripts/pkg.in:5]: Use an `if`  condition instead, with double quotes surrounding the sub shell.

Comment (Damaj301damaj-lol) [reply scripts/pkg.in]: Done, review lasted commit please

**Comment by Damaj301damaj-lol** [comment]
> That gets engaged way too late and not for `pacman`. 
Actually it does engage in pacman https://github.com/termux/termux-packages/pull/14291/commits/84d26e65876700f5872b0416f64bea1bf8c431ce

**Comment by agnostic-apollo** [comment]
>Actually it does engage in pacman

Ah, right, only added a month ago.

Thanks.

**Pull request merged**